microsoft-edge/devtools-guide-chromium/console/index.md

```diff
@@ -46,7 +46,7 @@ The fastest way to directly open the **Console** is to select `Control`+`Shift`+
 
 ## Look up error messages on the web from the Console
 
-Search the web for your **Console** error strings, right from within DevTools.  In the **Console**, an error message has a **Search for this message on the Web** button, shown as a magnifying glass.  When you select the **Search for this message on the Web** button, a new tab opens in the browser and shows search results for the error string.
+Search the web for your **Console** error strings, right from within DevTools.  In the **Console**, many error messages have a **Search for this message on the Web** button, shown as a magnifying glass.  When you select the **Search for this message on the Web** button, a new tab opens in the browser and shows search results for the error string.
```

Comment:
I'd like us to be consistent about `error strings` or `error messages` or just `errors`. We should just pick one and use it throughout. I'd recommend `errors` because I like being concise

Reply:
"Errors" works for me